internal/distributed/streaming/streaming.go

```diff
@@ -19,8 +23,8 @@ func Init(c *clientv3.Client) {
 
 // Release releases the resources of the wal accesser.
 func Release() {
-	if singleton != nil {
-		singleton.Close()
+	if w, ok := singleton.(*walAccesserImpl); ok && w != nil {
```

Comment:
Suggest adding a close method to the WALAccesser interface and invoking it directly.